src/pages/nerd-bytes.js

```diff
@@ -0,0 +1,153 @@
+import Video from '../components/Video';
+
+const NerdbytesPage = () => {
+  return (
+    <PageContext.Provider>
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="New Relic Nerd Bytes" />
+
+        <PageLayout.Content>
+          <section
+            css={css`
+              margin-top: 0;
+            `}
+            className={cx(styles.intro, 'intro-text')}
+          >
+            <div className={styles.introText}>
+              <p>
+                <Link
+                  className={styles.externalLink}
+                  to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+                >
+                  Nerd Bytes is a video podcast series
+                  <FeatherIcon
+                    className={styles.externalLinkIcon}
+                    name="external-link"
+                  />
+                </Link>{' '}
+                produced by engineers and practitioners <em>for</em> engineers
+                and practitioners.
+                <br />
+                <br />
+                Nerd Bytes episodes are crowd sourced, super short, micro topic
+                focused video screencast demos of tips, tricks, features,
+                functionality, news and ideas. Less formal than documentation,
+                easier to digest than a tutorial. Released regularly to keep New
+                Relic practitioners up to date with fresh and interesting
+                content.
+              </p>
+              <p>
+                <Button
+                  as={Link}
+                  variant={Button.VARIANT.PRIMARY}
+                  to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+                >
+                  Nerd Bytes playlist
+                </Button>
+              </p>
+            </div>
+            <div
+              css={css`
+                flex: 1;
+                margin-top: 0;
+                width: 100%;
+              `}
+            >
+              <Video id="sN5GpX0eWMc" type="youtube" />
+            </div>
+          </section>
+
+          <section>
+            <h2>How to consume Nerd Bytes</h2>
+            <p>
+              Find the latest Nerd Bytes episode on the New Relic YouTube
+              channel{' '}
+              <Link
+                className={styles.externalLink}
+                to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+              >
+                Nerd Bytes Playlist
+                <FeatherIcon
+                  className={styles.externalLinkIcon}
+                  name="external-link"
+                />
+              </Link>
+              . But, we encourage you to subscribe to the Nerd Bytes RSS feed
+              and add it to your Slack or Team channel. Fresh content will be
+              delivered directly to you when it is published.{' '}
```

Comment:
is this {' '} required here?

Reply:
Space wasn't needed and has been removed.